bpdm-gate/src/main/kotlin/org/eclipse/tractusx/bpdm/gate/config/PoolConfigProperties.kt

```diff
@@ -25,4 +25,5 @@ import org.springframework.boot.context.properties.ConfigurationProperties
 @ConfigurationProperties(prefix = "bpdm.pool")
 data class PoolConfigProperties(
     val baseUrl: String = "http://localhost:8080/api/catena",
```

Comment:
Base-URL is without api/catena

Reply:
i used the same value  that already exists on application.properties , should i need to change it on application.properties too?

Reply:
Yes please, the Gate was connected to the Pool before. Seems to be that was an outdated property.